contracts/protocol/facets/SellerHandlerFacet.sol

```diff
@@ -209,110 +205,134 @@ contract SellerHandlerFacet is SellerBase {
 
         require(exists, NO_PENDING_UPDATE_FOR_ACCOUNT);
 
-        // Get storage location for seller
-        (, Seller storage seller, AuthToken storage authToken) = fetchSeller(_sellerId);
+        {
```

Comment:
Maybe add a comment that this block solves for a stack to deep error.

Reply:
actually there is no need for this block, I fixed the stack to deep error removing local `sender` variable.

Reply:
Then why is this code still in a block? I checked git blame on the block opening curly and it was from a commit that said stack to deep error. Is there a reason to keep this in a block still?